xml/System.Runtime.Intrinsics.X86/Avx2.xml

```diff
@@ -3617,7 +3617,17 @@
       </Parameters>
       <Docs>
         <param name="address">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>VPMOVZXBW ymm, m128</summary>
+        <returns>To be added.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The native signature does not exist. We provide this additional overload for completeness.
```

Comment:
Yes, I believe they will. The point is that, while most of these map to "native" C++ intrinsics, some do not. For example, these take an address (pointer), since the instruction supports an in-memory operand. The C++ intrinsics don't provide those as distinct overloads.